### ISSUE TITLE
Can't print expressions in Observatory when in an async function

### ISSUE THREAD

**Issue by johnpryan** [dart-lang/sdk#27448]
**Steps to reproduce:**

1. run the following script using `pub run test -p content-shell --pause-after-load`
2. set breakpoints in each test after the variable `s` is assigned
3. observe that evaluating `print s` throws an error

**example code:** 

```dart
main() {
  group("observatory", () {
    test("sync", () {
      var s = "foo";
      s = "bar";
    });
    test("async", () async {
      var s = "foo";
      s = "bar";
    });
  });
}
```

**exception:**
```
$ p s
'evaluate:source': error: line 1 pos 2: parameter name or type expected
```

**Comment by rmacnak-google** [comment]
What version are you running? This seems like a duplicate of #27167.

**Comment by johnpryan** [comment]
I'm running 1.19.1 - feel free to close if this is fixed in 1.20

**Comment by rmacnak-google** [comment]
Fix (c80b23b7ff87a6fa084a4c218d6339462467b369) is in 1.20.
